docs/stdlib/array.rst

```diff
@@ -297,9 +294,7 @@ Reference
 
     :index: fill
 
-    Make a new array of specified size and filled with specified value.
-
-    Create anarray of size *n* where every element has the value *val*.
+    Returns a new array of size ``n`` with the specified ``val``:
```

Comment:
Since the one-liners are recycled in many cases for the tables at the top, we want to globally ensure none of them ends with a colon so that it doesn't make it into that table. A period, even before the code sample would be fine.

Reply:
Learned from @vpetrovykh that we have a style of styling parameters by surrounding them with `*`. Also, since they will also be rendered outside the context of this (without the function signature) using `n` and `val` here will lose meaning.

```suggestion
    Returns a new array of a specified number of copies of the specified value.
```

Reply:
To avoid an aggressive amount of suggested changes, I'll go ahead and look through all of our functions and get these reverted, thank you for the heads up. I chose to code-format it instead of `*` so it'd look similar to the way the client renders it. Would it be possible if we could have the original syntax color format it as green like it is in the argument signature so it stands out a little better? I think this would be a great benefit

Reply:
@i0bs You're asking if emphasis (i.e., `*something*`) can be styled in green so that those references will be a consistent color? Confirming my understanding.

Reply:
Yep! Maybe if we color code to the format we can make it work better with the style we're wanting to follow here. Since we're doing this to reference arguments in the signature, we should try and respect that style heavily. Italics alone can be mixed with other italicized comments